### PR TITLE
Update PlotMesh.m

### PR DESCRIPTION
**Pull request by meluxmeme**
added the option to choose colormap.

### CODE REVIEW

File: PlotMesh.m [hunk 1]
```diff
@@ -1,27 +1,27 @@
 # matlab codes to plot mesh
-function PlotMesh(coordinates,nodes)
-nel = length(nodes) ;                  % number of elements
-nnode = length(coordinates) ;          % total number of nodes in system
+function PlotMesh(coordinates,nodes,colors)
+Xnodes= length(nodes) ;                  % number of elements
+Ynodes = length(coordinates) ;          % total number of nodes in system
```
Comment (baraltalaz) [inline PlotMesh.m:4]: add color option

File: PlotMesh.m [hunk 2]
```diff
@@ -1,27 +1,27 @@
...
 nnel = size(nodes,2);                % number of nodes per element
 % 
 % Initialization of the required matrices
+nel=Xnodes;
+nnel=Ynodes;
```
Comment (baraltalaz) [inline PlotMesh.m:9]: edit it

File: PlotMesh.m [hunk 3]
```diff
@@ -1,27 +1,27 @@
...
-end
-    
+end   
 % Plotting the FEM mesh, diaplay Node numbers and Element numbers
      f1 = figure ;
      set(f1,'name','Mesh','numbertitle','off') ;
      plot(X,Y,'k')
      %fill(X,Y,'w')
-     colormap(jet);
+     colormap(colors);
```
Comment (baraltalaz) [inline PlotMesh.m:24]: add

**Review by baraltalaz — COMMENTED** [review]
ok

**Comment by baraltalaz** [comment]
merged

**Review by meluxmeme — COMMENTED** [review]
got it

**Pull request merged**